From d6647a11cb9a31fc8706018865fc43a1b4ea8296 Mon Sep 17 00:00:00 2001
From: Lrs121 <lrs@lrsservers.net>
Date: Thu, 30 Jun 2016 11:37:54 -0700
Subject: [PATCH] Patch battery values

Pulls the values from qpnp-adc-common-sparrow.h
and inserts them in line

Credit to XDA User T10NAZ

Upstream-Status: Inappropriate
---
 drivers/hwmon/qpnp-adc-common.c | 95 +++++++++++++++++++++++++++++++++++++++++
 1 file changed, 95 insertions(+)

diff --git a/drivers/hwmon/qpnp-adc-common.c b/drivers/hwmon/qpnp-adc-common.c
index e93bbbf5d466..b49f3d31a26b 100644
--- a/drivers/hwmon/qpnp-adc-common.c
+++ b/drivers/hwmon/qpnp-adc-common.c
@@ -54,6 +54,101 @@
    their framework which is 0.1DegC. True resolution of 0.1DegC
    will result in the below table size to increase by 10 times */
 
+static const struct qpnp_vadc_map_pt adcmap_btm_threshold[] = {
+        {-300,  1520},
+        {-200,  1520},
+        {-100,  1369},
+        {-90,   1356},
+        {-80,   1344},
+        {-70,   1331},
+        {-60,   1318},
+        {-50,   1305},
+        {-40,   1290},
+        {-30,   1277},
+        {-20,   1264},
+        {-10,   1250},
+        {0,     1236},
+        {10,    1251},
+        {20,    1238},
+        {30,    1225},
+        {40,    1212},
+        {50,    1199},
+        {60,    1186},
+        {70,    1173},
+        {80,    1160},
+        {90,    1147},
+        {100,   1122},
+        {110,   1111},
+        {120,   1097},
+        {130,   1086},
+        {140,   1074},
+        {150,   1060},
+        {160,   1048},
+        {170,   1035},
+        {180,   1021},
+        {190,   1009},
+        {200,   998},
+        {210,   987},
+        {220,   977},
+        {230,   966},
+        {240,   952},
+        {250,   943},
+        {260,   932},
+        {270,   923},
+        {280,   911},
+        {290,   902},
+        {300,   891},
+        {310,   881},
+        {320,   871},
+        {330,   860},
+        {340,   849},
+        {350,   839},
+        {360,   832},
+        {370,   823},
+        {380,   813},
+        {390,   807},
+        {400,   801},
+        {410,   790},
+        {420,   779},
+        {430,   774},
+        {440,   769},
+        {450,   760},
+        {460,   750},
+        {470,   742},
+        {480,   738},
+        {490,   730},
+        {500,   721},
+        {510,   714},
+        {520,   713},
+        {530,   707},
+        {540,   701},
+        {550,   695},
+        {560,   689},
+        {570,   683},
+        {580,   677},
+        {590,   671},
+        {600,   665},
+        {610,   659},
+        {620,   653},
+        {630,   647},
+        {640,   641},
+        {650,   635},
+        {660,   629},
+        {670,   623},
+        {680,   617},
+        {690,   611},
+        {700,   496},
+        {710,   490},
+        {720,   484},
+        {730,   478},
+        {740,   472},
+        {750,   466},
+        {760,   460},
+        {770,   454},
+        {780,   448},
+        {790,   442}
+};
+
 static const struct qpnp_vadc_map_pt adcmap_qrd_btm_threshold[] = {
 	{-200,	1540},
 	{-180,	1517},
